src/ProtoValidate/Internal/Evaluator/EvaluatorBuilder.cs

```diff
@@ -144,6 +144,9 @@ private void ProcessOneofConstraints(MessageDescriptor desc, MessageEvaluator ms
         foreach (var oneofDesc in oneofs)
         {
             var oneofConstraints = ConstraintResolver.ResolveOneofConstraints(oneofDesc);
+
+            if (oneofConstraints is null) continue;
```

Comment:
The short-circuiting with the null check here causes a number of conformance tests to fail.